Formula/libxmlsec1.rb

```diff
@@ -29,6 +29,10 @@ class Libxmlsec1 < Formula
     depends_on xcode: :build
   end
 
+  on_linux do
+    depends_on "libxslt"
+  end
```

Comment:
I would expect this to be used on macOS too, so then it should be
```suggestion
  uses_from_macos "libxslt"
```